spec/factories/inspection.rb

```diff
@@ -1,5 +1,7 @@
 FactoryGirl.define do
   factory :inspection do
+    person
+    item
```

Comment:
I would prefer not to use this style. I have found it results in cascading lines of objects bring created. Imagine if the person factory then has a reference to an event, that then has take and assignments and so on. I prefer to either create them explicitly in the spec, or stub them.
Thanks for your help